pandas/core/generic.py

```diff
@@ -175,7 +175,10 @@ def is_copy(self, msg):
     def _validate_dtype(self, dtype):
         """ validate the passed dtype """
 
-        if dtype is not None:
+        # GH 26336: don't convert 'category' to CategoricalDtype
+        if isinstance(dtype, str) and dtype == 'category':
```

Comment:
Using this pattern here and in a few other places to address https://github.com/pandas-dev/pandas/issues/26336#issuecomment-494230444.  Could maybe push this back into `pandas_dtype`?  Can investigate that further if that'd be preferable to this special casing.